digital_benchmark/digital_benchmark/settings.py

```diff
@@ -118,3 +119,11 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# facebook_benchmark settings
+facebook_graph_api_version = '3.1'
+facebook_default_fields_for_page = 'id,displayed_message_response_time,engagement{count},fan_count,name,overall_star_rating,rating_count,talking_about_count,unread_message_count,unread_notif_count,unseen_message_count,verification_status'
```

Comment:
Can these be in an array?

Reply:
we have to send them as comma separated string to graph api
if we use them as array/list them we can join them with string join function

Reply:
Make them a list in the settings. We can join them where we use them. Pl discuss this with me tomorrow.